sys/vane/clay.hoon

```diff
@@ -3576,19 +3703,33 @@
     |=  {pot/term bem/beam}
     [u.hez.ruf %give %ogre pot]
   ::
-      $warp
+      $perm
+    =^  mos  ruf
+      ::TODO  after new boot system, just use our from global.
+      =+  den=((de now hen ruf) [. .]:our.q.hic des.q.hic)
+      abet:(perm:den pax.q.hic rit.q.hic)
+    [mos ..^$]
+  ::
+      ?($warp $werp)
+    =^  for  q.hic
+      ?:  ?=($warp -.q.hic)
+        [~ q.hic]
+      :_  [%warp q.q.hic r.q.hic]
+      ?:  =(p.q.hic p.q.q.hic)  ~&([%huh-this-west-may-be-weird p.q.hic] ~)
```

Comment:
What's weird about it?

Reply:
Nothing but the fact that I left a debug statement in it. Cleaning up.